.github/workflows/vm-auto-shutdown.yaml

```diff
@@ -2,7 +2,7 @@ name: vm-auto-shutdown
 on:
   workflow_dispatch:
   schedule:
-    - cron: '0 19,22 * * *' # Every day at 20:00 and 23:00 BST
+    - cron: "0 19,22 * * *" # Every day at 20:00 and 23:00 BST
```

Comment:
Whats the reason for changing to double quotes? Keen to keep consistency with the other flows. Suggest you change the rest as well if there is benefit?